Makefile

```diff
@@ -17,10 +18,14 @@ run-build:
 	@${ps-bundle} && BROWSER_RUNTIME=1 webpack --mode=production
 
 check-format:
-	@purs-tidy check ${ps-sources} && nixpkgs-fmt --check ${nix-sources}
+	@purs-tidy check ${ps-sources}
+	nixpkgs-fmt --check ${nix-sources}
+	fourmolu -m check -o -XTypeApplications -o -XImportQualifiedPost ${hs-sources}
```

Comment:
I think the `at` symbol is to quiet the command, perhaps you want to do that as well for all of them?